src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -79,21 +79,6 @@ def transform_vm_list(vm_list):
     return [transform_vm(v) for v in vm_list]
 
 
-def transform_av_set_output(av_set):
```

Comment:
Can this actually be removed?  Isn't it still needed for any API version prior to 2017-04-30 (presumably stack?)